google/cloud/ndb/key.py

```diff
@@ -739,6 +739,29 @@ def urlsafe(self):
         raw_bytes = self.serialized()
         return base64.urlsafe_b64encode(raw_bytes).strip(b"=")
 
+    def to_legacy_urlsafe(self, location_prefix=None):
+        """A ``Reference`` protobuf serialized and encoded as urlsafe base 64.
```

Comment:
Docstring suggestion:

        """A urlsafe serialized ``Reference`` protobuf with an App Engine prefix

        This will produce a urlsafe string which includes an App Engine location prefix ("partition"), compatible with the Google Datastore admin console.

        Arguments:
            location_prefix (str): A location prefix ("partition") to be prepended to the key's `project` when serializing the key. A typical value is "s~", but "e~" or other partitions are possible depending on the project's region and other factors.